P5/Source/Specs/TEI.xml

```diff
@@ -199,6 +198,7 @@ $Id$
     TEI namespace <code>http://www.tei-c.org/ns/1.0</code> on it, using
     the <att>xmlns</att> attribute.</p>
   </remarks>
+  <remarks versionDate="2022-02-20" xml:lang="es"><p>El elemento es obligatorio. Normalmente se especifica el espacio de nombre TEI en su interior <code>http://www.tei-c.org/ns/1.0</code>, utilizando el atributo <att>xmlns</att></p></remarks>
```

Comment:
I think the most common term is Spanish (hopefully not just in Peninsular Spanish) is “espacio de nombres” (my suggestion here would be: “el espacio de nombres de TEI”)

Reply:
I think the MDN can be consider an authority source. However, to add a a reference in support of “espacio de nombres” instead, there is the [Spanish Wikipedia article](https://es.wikipedia.org/wiki/Espacio_de_nombres) which provides a couple of other references in Spanish that use it.

Reply:
That's fair. In that case, I'd recommend glossing it the other way, like Wikipedia does "espacio de nombres (namespace)".

Reply:
See #2236.

Reply:
> 
Having a feature to see comments would be great, but with some of my students were talking that having a controlled vocabulary is quite important... (don't know in which state remained Raff spreadsheet) (like, is it "espacio de nombres" or "espacio de nombre", header is "encabezado" o "cabecera", etc.).

Reply:
Most probably you are right @HelenaSabel ;) In my case I always used it in singular since we normally use one name space (but we can have multiples).